### ISSUE TITLE
SPC not saving absorbance curve?

### ISSUE THREAD

**Issue by mzieg** [WasatchPhotonics/ENLIGHTEN#271]
Customer report indicates SPC output may not be correctly capturing processed absorbance. Also check SPC enums for non-Raman techniques.

**Comment by samiebee43** [comment]
duplicate of #276 ?

**Comment by mzieg** [comment]
I don't think this is related to #276.  We are talking with a potential new partner company who will help us better understand our .spc file export however, and will perhaps give us tools to let us better diagnose this issue.
